docs/index.md

```diff
@@ -38,41 +38,39 @@ resource "ec_deployment" "my_deployment" {
 
 ## Authentication
 
-The Elastic Cloud provider offers two methods of authentication against the remote API. Depending on the target environment, one or all can be used. By default the `endpoint` which the provider will target is the Public API of the Elasticsearch Service (ESS) offering.
+The Elastic Cloud (EC) provider offers two methods of authentication against the remote API: `apikey` or a combination of `username` and `password`. Depending on the environment, one or both can be used. The Public API of Elasticsearch Service (ESS) is the default `endpoint` that the provider will target.
 
-Only one of `apikey` or `username` and `password`  can be specified at one time. The Elasticsearch Service (ESS) offering, only supports API Keys as the authentication mechanism. When targeting an ECE
-Installation, `username` and `password` can be used.
+Elasticsearch Service (ESS) only supports `apikey`. Elastic Cloud Enterprise (ECE) supports both `apikey` and a combination of `username` and `password`.
 
 !> **Warning:** Hard-coding credentials into any Terraform configuration is not
 recommended, and risks secret leakage should this file ever be committed to a
 public version control system.
 
-### API Key authentication (Recommended)
+### API Key authentication (recommended)
```

Comment:
```suggestion
### API key authentication (recommended)
```

AFAIK, `key` is not a proper noun here, based on how we refer to API keys [in our docs](https://www.elastic.co/guide/en/cloud/current/ec-api-authentication.html).